src/components/Authentication.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { useState , useEffect} from 'react'
+import Navbar from './Navbar';
+
+
+export default function Authentication() {
+    const [email,setEmail]=useState("");
+    const [password,setPassword]=useState("");
+    const [confirmPassword,setConfirmPassword]=useState("");
+    const [isChecked,setIsChecked]=useState(false);
+    const [isCheckedRegister,setIsCheckedRegister]=useState(false);
+    const [isRegistered,setIsRegistered]=useState(false);
+    const [validationMessage,setValidationMessage]=useState("");
+
+    let handleCheck=()=>{
+        setIsChecked(!isChecked);
+    }
+
+    let handleCheckRegister=()=>{
+        if(!isCheckedRegister){
+            //check email is valid
+            if(password == confirmPassword){
+                setIsCheckedRegister(!isCheckedRegister);
+                setValidationMessage("");
+            }else{
+                setValidationMessage("Password and Confirm Password doesn't match");
+            }
+        }else{
+            setIsCheckedRegister(!isCheckedRegister);
+        }
+    }
+
+    return (
+        <div>
+            <div class="login-form">
+            {
+                !isRegistered?
```

Comment:
No need to use  ternary operator. You can directly use && for the div's which are required